MdePkg/MdePkg.ci.yaml

```diff
@@ -78,6 +78,7 @@
             "Include/Protocol/DebugSupport.h",
             "Test/UnitTest/Library/BaseSafeIntLib/TestBaseSafeIntLib.c",
             "Library/BaseFdtLib",
+            "Library/CompilerIntrinsicsLib",

```

Comment:
Does this need to be ignored here? Seemingly it wasn’t failing here, just on the ArmPkg side because it wasn’t found?

Reply:
It was added to ArmPkg.ci.yaml in the following commit:
```
commit 365dced2c37a55914d2e8e57b39288369c94e949
Author: Michael D Kinney <michael.d.kinney@intel.com>
Date:   Wed Nov 17 09:18:53 2021 -0800

    ArmPkg: Update YAML to ignore specific ECC files/errors
    
    REF: https://bugzilla.tianocore.org/show_bug.cgi?id=3749
    
    Update package YAML files to ignore ECC errors that are
    already present.  These issues must be fixed in the future,
    but should not block source code changes for these known
    issues.
```

I presume those issues are still present in the copy that's now in MdePkg.

Reply:
Sure, but my point is that MdePkg CI has been passing, so seemingly ECC has not been having issues with it

Reply:
Oh, thanks! I understand now. I've updated the PR to just remove it from ArmPkg.ci.yaml.